integrations/chroma/tests/test_document_store.py

```diff
@@ -406,18 +406,6 @@ def test_nested_logical_filters(self, document_store: ChromaDocumentStore, filte
             ],
         )
 
-    @pytest.mark.skip(reason="Filter on dataframe contents is not supported.")
-    def test_comparison_equal_with_dataframe(
-        self, document_store: ChromaDocumentStore, filterable_docs: List[Document]
-    ):
-        pass
-
-    @pytest.mark.skip(reason="Filter on dataframe contents is not supported.")
-    def test_comparison_not_equal_with_dataframe(
-        self, document_store: ChromaDocumentStore, filterable_docs: List[Document]
-    ):
-        pass
-
```

Comment:
Chroma does not support dataframes, so in this case I'm not using the new testing class.
It is no longer necessary to skip these tests: they are no longer part of the basic tests.